intbot/conftest.py

```diff
@@ -17,3 +20,66 @@ def github_data():
             open(base_path / "query_result.json"),
         )["data"]["node"],
     }
+
```

Comment:
This is just moved to a global conftest from a different file - since now we have more than one file with async tests.